.github/workflows/haxe-language-server.yml

```diff
@@ -0,0 +1,43 @@
+name: Package haxe-language-server
+
+on:
+  workflow_dispatch:
+    inputs:
+      haxe-language-server_version:
+        type: string
+        required: false
+        default: "1.0.0"
+
+jobs:
+  package_all:
+    name: Package haxe-language-server
+    runs-on: ubuntu-latest
+    permissions: write-all
+    env:
+      VERSION: ${{ inputs.haxe-language-server_version }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Update Packages
+        run: sudo apt-get update -y
+      - name: Install Dependencies
+        run: sudo apt-get install -y npm nodejs git
+      - name: Get haxe-language-server
+        run: git clone https://github.com/vshaxe/haxe-language-server
```

Comment:
Looks like they don't have any tags, right? How should we version this?
We could use their main repo https://github.com/vshaxe/vshaxe, clone it at some tag and use their `server` subdir.

Reply:
Ok. I'll do it

Reply:
Now this PR contains only packaging (based on versioned vshaxe)